Prog/BPK Summary Tabs_2013-17.sas

```diff
@@ -0,0 +1,566 @@
+				r_mprice_condo_2001 r_mprice_condo_2002 r_mprice_condo_2003 r_mprice_condo_2004
+				r_mprice_condo_2005 r_mprice_condo_2006 r_mprice_condo_2007 r_mprice_condo_2008
+				r_mprice_condo_2009 r_mprice_condo_2010 r_mprice_condo_2011 r_mprice_condo_2012
+				r_mprice_condo_2013 r_mprice_condo_2014 r_mprice_condo_2015 r_mprice_condo_2016
+				r_mprice_condo_2017 r_mprice_condo_2018
+
+				r_mprice_tot_2000
+				r_mprice_tot_2001 r_mprice_tot_2002 r_mprice_tot_2003 r_mprice_tot_2004
+				r_mprice_tot_2005 r_mprice_tot_2006 r_mprice_tot_2007 r_mprice_tot_2008
+				r_mprice_tot_2009 r_mprice_tot_2010 r_mprice_tot_2011 r_mprice_tot_2012
+				r_mprice_tot_2013 r_mprice_tot_2014 r_mprice_tot_2015 r_mprice_tot_2016
+				r_mprice_tot_2017 r_mprice_tot_2018	)
+
+		Police.Crimes_sum_&geosuf
+			(keep=&geo 
+				crimes_pt1_property_2000 crimes_pt1_property_2001 
+				crimes_pt1_property_2002 crimes_pt1_property_2003 crimes_pt1_property_2004 
+				crimes_pt1_property_2005 crimes_pt1_property_2006 crimes_pt1_property_2007 
+				crimes_pt1_property_2008 crimes_pt1_property_2009 crimes_pt1_property_2010
+				crimes_pt1_property_2011 crimes_pt1_property_2012 crimes_pt1_property_2013
+				crimes_pt1_property_2014 crimes_pt1_property_2015 crimes_pt1_property_2016 
+				crimes_pt1_property_2017 
+				crimes_pt1_violent_2000 crimes_pt1_violent_2001 
+				crimes_pt1_violent_2002 crimes_pt1_violent_2003 crimes_pt1_violent_2004 
+				crimes_pt1_violent_2005 crimes_pt1_violent_2006 crimes_pt1_violent_2007 
+				crimes_pt1_violent_2008 crimes_pt1_violent_2009 crimes_pt1_violent_2010 
+				crimes_pt1_violent_2011 crimes_pt1_violent_2012 crimes_pt1_violent_2013 
+				crimes_pt1_violent_2014 crimes_pt1_violent_2015 crimes_pt1_violent_2016 
+				crimes_pt1_violent_2017
+				crime_rate_pop_2000 crime_rate_pop_2001 crime_rate_pop_2002 crime_rate_pop_2003 
+				crime_rate_pop_2004 crime_rate_pop_2005 crime_rate_pop_2006 crime_rate_pop_2007 
+				crime_rate_pop_2008 crime_rate_pop_2009 crime_rate_pop_2010 crime_rate_pop_2011 
+				crime_rate_pop_2012 crime_rate_pop_2013 crime_rate_pop_2014 crime_rate_pop_2015 
+				crime_rate_pop_2016 crime_rate_pop_2017)
+		
+		NCDB.Ncdb_sum_&geosuf
+			(keep= &geo totpop_1990 totpop_2000 
+				numoccupiedhsgunits_1990 numoccupiedhsgunits_2000)
+
+		NCDB.Ncdb_sum_2010_&geosuf
+			(keep= &geo totpop_2010 numoccupiedhsgunits_2010); 
+
+		by &geo;
+
+		/*Labor force, employment, and poverty*/
+			PctLaborForce_&_years. = popincivlaborforce_&_years. / pop16andoveryears_&_years.;
+			PctPopEmployed_&_years. = popcivilianemployed_&_years. / popincivlaborforce_&_years.;
+			PctPopUnemployed_&_years. = popunemployed_&_years. / popincivlaborforce_&_years.;
+			PctPoorPersons_&_years. = poppoorpersons_&_years. / personspovertydefined_&_years.;
+			PctPoorChildren_&_years. = poppoorchildren_&_years. / childrenpovertydefined_&_years.;
+
+		/*Employment by major occupations*/
+			PctEmpMngmt_&_years. = popemployedmngmt_&_years. / popemployedbyocc_2012_16;
```

Comment:
The denominators for the employment by occupation vars need to be updated. In fact, it would be better not to hard code the years and use the _&\_years._ macro variable instead.

Note: Two ways that this problem revealed itself. First, there were uninitialized variable notes in the LOG. 
`NOTE: Variable popemployedbyocc_2012_16 is uninitialized.`

Second, the values of the PctEmp vars were all missing in the Proc Means output. 
```
The MEANS Procedure

Variable                            N             Sum            Mean         Std Dev         Minimum         Maximum
---------------------------------------------------------------------------------------------------------------------
PctEmpMngmt_2013_17                 0               .               .               .               .               .
popemployedbyocc_2012_16            0               .               .               .               .               .
PctEmpNatRes_2013_17                0               .               .               .               .               .
PctEmpProd_2013_17                  0               .               .               .               .               .
PctEmpSales_2013_17                 0               .               .               .               .               .
PctEmpServ_2013_17                  0               .               .               .               .               .
```